benefits/enrollment/templates/enrollment/index--calfresh.html

```diff
@@ -1,14 +1,7 @@
 {% extends "enrollment/index.html" %}
 {% load i18n %}
 
-{% block media-items %}
-  <div class="col-12 col-sm-12 col-lg-8">
-    <ul class="mb-5 mx-0 px-0 d-flex list-unstyled justify-content-center flex-column">
-      {% include "enrollment/includes/media-item--contactless-card--index--calfresh.html" %}
-    </ul>
-  </div>
-{% endblock media-items %}
-
-{% block additional-info %}
-  <div class="col-12 col-sm-12 col-lg-8">{% include "enrollment/includes/alert-box--warning--calfresh.html" %}</div>
-{% endblock additional-info %}
+{% block info %}
+  {% include "enrollment/includes/info--contactless-card--index--calfresh.html" %}
+  {% include "enrollment/includes/alert-box--warning--calfresh.html" %}
+{% endblock info %}
```

Comment:
Taking suggestions for this naming. @thekaveman @angela-tran Really wasn't sure what to name the block (`info`) or the includes. I could even get rid of the includes and just copy/paste the h2/p/modal in directly. Thoughts?

Reply:
Done by @machikoyasuda  in [56b9322](https://github.com/cal-itp/benefits/pull/2575/commits/56b9322b3bc829d78341e21a96555a1e9cdb89d1)